app.py

```diff
@@ -0,0 +1,131 @@
+import os
+import json
+import requests
+from flask import Flask, redirect, request
+from typing import Dict
+
+
+REDCAP_API_TOKEN = os.environ['REDCAP_API_TOKEN']
+REDCAP_API_URL = os.environ['REDCAP_API_URL']
+IS_COMPLETE = 1
+app = Flask(__name__)
+
+
+def fetch_net_ids() -> Dict[str, str]:
+    """
+    Fetches all the existing NetIDs in the given REDCap project.
```

Comment:
I have a question about retrieving all the REDCap records from the project every time the Musher gets invoked. See https://seattle-flu-study.slack.com/archives/C0173B4S1LY/p1599676988404600

Reply:
I agree, I don't think we need to fetch all records. 
The REDCap API has a `filterLogic` field that we can use to try to fetch the one record that has matching netid

Reply:
Thanks, both. I've pushed a new commit that uses REDCap-side filtering so I only fetch one record at a time.

Reply:
Looks like the docstring still needs to be update here

Reply:
Fixed